target/linux/mvebu/cortexa53/base-files/lib/upgrade/platform.sh

```diff
@@ -23,8 +24,10 @@ platform_check_image() {
 
 platform_do_upgrade() {
 	case "$(board_name)" in
+	globalscale,ccpe|\
```

Comment:
Doesn't belong here?

Reply:
sorry, CCPE was originalyl used as a code name and I managed to overlook this one while cleaning. Fixed now

Reply:
Still not fixed.